src/screen/EnhanceScreen_recovery.java

```diff
@@ -0,0 +1,112 @@
+package screen;
+
+import java.awt.event.KeyEvent;
+
+import engine.Cooldown;
+import engine.Core;
+import engine.SoundEffect;
+
+public class EnhanceScreen_recovery extends Screen {
```

Comment:
Codacy Static Code Analysis — 3 new issues (0 max.) of at least minor severity. 

1. `MINOR` `Code Style` The class name 'EnhanceScreen_recovery' doesn't match '[A-Z][a-zA-Z0-9]*' 
It does not conform to Java's naming convention. You can fix the name like this:
```
public class EnhanceScreenRecovery extends Screen {
```